stubs/workalendar/workalendar/usa/wyoming.pyi

```diff
@@ -0,0 +1,4 @@
+from .core import UnitedStates
+
+class Wyoming(UnitedStates):
+    martin_luther_king_label: str
```

Comment:
Presumably most attributes should be e.g. `ClassVar[str]` instead of `str`, since the library does this:

```
@iso_register('US-WY')
class Wyoming(UnitedStates):
    """Wyoming"""
    martin_luther_king_label = "Martin Luther King, Jr. / Wyoming Equality Day"
```

This applies to a lot of other attributes, so I wouldn't change them all by hand. For example, you could use a script, a `sed` command, or a fancy editor.

Reply:
I pushed an improved version just now. thanks for reviewing